app/lib/ui/main_screen.dart

```diff
@@ -40,7 +40,10 @@ class _MainScreenState extends State<MainScreen> {
 
     return Consumer<ModelTheme>(builder: (context, ModelTheme themeNotifier, child) {
       return Scaffold(
-          appBar: MainAppBar(selectedItem.label),
+          appBar: AppBar(
+            title: Text(selectedItem.label),
+            actions: [ThemeSelector()],
+          ),
```

Comment:
Why did you remove the `MainAppBar`? This is the factorized top app bar, I think we have to keep it.